staging/src/k8s.io/cloud-provider/app/controllermanager.go

```diff
@@ -113,8 +122,11 @@ the cloud specific control loops shipped with Kubernetes.`,
 
 	fs := cmd.Flags()
 	namedFlagSets := s.Flags(ControllerNames(controllerInitFuncConstructors), ControllersDisabledByDefault.List())
-	verflag.AddFlags(namedFlagSets.FlagSet("global"))
-	globalflag.AddGlobalFlags(namedFlagSets.FlagSet("global"), cmd.Name())
+
+	globalFlagSet := namedFlagSets.FlagSet("global")
+	verflag.AddFlags(globalFlagSet)
+	logsapi.AddFlags(logOptions, globalFlagSet)
```

Comment:
In general, I would like the logging options here to be consistent with kube-controller-manager and this seems to deviate from this: https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-controller-manager/app/controllermanager.go#L155-L162

Reply:
Ah, actually we are still doing this here: https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-controller-manager/app/options/options.go#L188

Reply:
so this can stay as-is or should I change something?

Reply:
I think it can stay as-is